collect_app/src/test/java/org/odk/collect/android/utilities/gdrive/GoogleAccountsManagerTest.java

```diff
@@ -66,24 +69,34 @@ private void stubAccountPermission(boolean permission) {
     }
 
     private void stubCredential() {
-        doAnswer(new Answer() {
-            @Override
-            public Object answer(InvocationOnMock invocation) throws Throwable {
-                currentAccount = invocation.getArgument(0);
-                return null;
-            }
+        doAnswer(invocation -> {
+            currentAccount = invocation.getArgument(0);
+            return null;
         }).when(mockedCredential).setSelectedAccountName(anyString());
+
+        stubAccount(EXPECTED_ACCOUNT);
+    }
+
+    private void stubAccount(String name) {
+        Account account = mock(Account.class);
+        Whitebox.setInternalState(account, "name", name);
+        doReturn(new Account[]{account}).when(mockedCredential).getAllAccounts();
+    }
+
+    private void removeAccounts() {
+        doReturn(new Account[]{}).when(mockedCredential).getAllAccounts();
```

Comment:
Method comment of `getAllAccounts()` says that 
```
 Returns all Google accounts or {@code null} for none. 
```

Reply:
So, returning null would be better in that case

Reply:
Yeah! Best to match the signature for sure.